userfaultfd-sys/wrapper.h

```diff
@@ -1 +1,4 @@
 #include <linux/userfaultfd.h>
+#ifndef UFFD_USER_MODE_ONLY
```

Comment:
please add a comment here indicating this definition is not available before linux 5.11, and is provided so that userfaultfd-sys has the same exports on all kernels